### DISCUSSION TITLE
New Course Idea: Communicating Complexity

### DISCUSSION THREAD

**Discussion by nathenf** [life-itself/community#210] [General]
The current crises we face is a complex topic and is particularly hard to explain to people (all of us in many respects) that lack the bigger picture of the situation. This shows up in:

- Trying to explain how two seemingly unrelated topics work symbiotically (for example mental health and farming)
- Persuading someone to make better decisions regarding daily actions to navigate the Polycrisis
- Explaining the Polycrisis in layman's terms
- Explaining what our organisations do and the issues they are dedicated to transforming

Communicating complexity would be an online program for people who want to improve their communication skills when discussing the crises, their complications and potential pathways for transformation. 

The course will include communication experts as course leaders, bringing their own insight and experience from years in on-the-ground and digital communication. 

@EilidhRoss1 to add your thoughts

**Comment by Alex-Goodall** [comment]
Hi Nathan

This seems to align quite closely with what I'm aiming for in [Relevant.Education](http://relevant.education) , and in the related book. 

Some conclusions and learnings from my cogitations over the last months - which I'm incorporating into the book, include:

- From my experience, people generally don't find it hard to break out of the linear-causation mindset intellectually, but of course that's very different to generating the embodied understanding which is the driver for action or personal change
- Related to that is the need to trigger emotions - which I'm sure the communication experts will agree on. Marketing 101 !
- A component in all this is the need to challenge assumptions at all levels: it's the, often unstated/unconscious, assumptions we make individually and collectively that act as the attractors in complexity. It is these assumption that need to evolve in order to effect systemic change.

Regards

Alex

**Reply by nathenf** [reply]
Thanks for sharing Alex,

Yes I do believe these two projects have similarity indeed. These are great points, and yes certainly emotional triggers and also personal relevance are key. 

I'm waiting for the book 😀

**Reply by rufuspollock** [reply]
> From my experience, people generally don't find it hard to break out of the linear-causation mindset intellectually, but of course that's very different to generating the embodied understanding which is the driver for action or personal change

such a great point @Alex-Goodall 

I think telling stories here could be one approach.

**Comment by rufuspollock** [comment]
Love the suggestion Nathen - thanks for sharing! I think finding accessible, compelling ways to communicate these interlinkages is indeed key.
